### DISCUSSION TITLE
Jump Menu for Control Panel Navigation

### DISCUSSION THREAD

**Discussion by acalvino4** [craftcms/cms#10551] [Ideas]
Something like the new jump menu in EE6 would be really slick to have in Craft's control panel by default:
https://www.youtube.com/watch?v=DqhPSPHPLc4&t=27s

There is a plugin that just came out that does this:
https://plugins.craftcms.com/palette

But two arguments for having this in core:
* I suspect the majority of users would appreciate it
* It could provide a standardized way for any plugins to add indexable entries/actions to the jump menu

**Comment by aaronbushnell** [comment]
FWIW I would be more than happy for the Pixel & Tonic team to swallow up [Palette](http://plugins.craftcms.com/palette) and make it part of core 😁
